distributed/stealing.py

```diff
@@ -148,7 +148,7 @@ def log(self, msg):
     def add_worker(self, scheduler=None, worker=None):
         self.stealable[worker] = tuple(set() for _ in range(15))
 
-    def remove_worker(self, scheduler=None, worker=None):
+    def remove_worker(self, scheduler: Scheduler, worker: str):
```

Comment:
mypy doesn't "prefer" this. It just defaults to `Any` if you don't specify anything.
What you ask exists - it's the ``--disallow-incomplete-defs`` option in mypy. However, enabling it would require a substantial (not unreasonable) amount of legwork. I started it in #6601 but it's going to be very low priority work.

Reply:
> mypy doesn't "prefer" this. It just defaults to Any if you don't specify anything.

Thanks. I thought the AST of a function w/out return statement would show this and figured mypy should be able to infer this. I may be wrong on both statements

> I started it in https://github.com/dask/distributed/pull/6601 but it's going to be very low priority work.

Yes, it would be nice to enable this. I would actually suggest to enable it file by file (or by module) to allow for faster merging, see https://mypy.readthedocs.io/en/stable/config_file.html#examples